src/components/widgets/AutocompleteWidget/AutocompleteWidget.tsx

```diff
@@ -120,11 +108,25 @@ function AutocompleteWidget(props: AutocompleteWidgetProps) {
                     colorFirst={"primary"}
                     colorSecond={"success"}
                   />
-                  {!singleSuggestionRow && value.description ?
-                    <>
+                  {showApiSource && value.source && value.source !== "" &&
+                    <span 
+                      className="euiBadge"
+                      style={{
+                        backgroundColor: "black", 
+                        color: "white", 
+                        display: "inline-block",
+                        marginLeft: "10px"                                  
+                      }}
+                    >                                                                                     
+                    {"Source: " + value.source}      
```

Comment:
Why is it necessary to see the full source API address at first glance?

Would it be more user friendly to just show OLS or SKOSMOS or ONTOPORTAL as the badge? The full source address is then shown in the hover.

Reply:
What is the correct badge to use is the question. Like "OLS" as the platform name or "TIB" and "ZBmed" as the provider organizations?

Reply:
yeah true. But leaving the end-user with the URL of the API doesn't make sense to me. Maybe both (provider and platform) are interesting, but in a presentation that is easy to read. Furthermore, a badge with a long address might lead to overflow of the line. We have use cases that need a small layout (e.g. https://github.com/ts4nfdi/terminology-service-suite/issues/98).

Reply:
I mean, this is a general point for the Gateway too. At the moment, it expects "ontoportal, ols, skosmos" as the databases. Is this the use-case? I think we should consider that maybe one wants to search based on the provider like zbmed. Do users care more about the software platform or the provider Org?

cc: @syphax-bouazzouni

Reply:
Maybe filtering should be allowed for both, provider and platform. Question is, what information do we present on the small space in the autocomplete drop down menu. I'm not sure anymore, if the user needs the software platform information at all. Also the BreadcrumbWidget, especially the short form (mostly a big number) may not be ideal. Maybe we should ask an end-user/create some use cases.